src/modes/install.ts

```diff
@@ -39,7 +46,30 @@ export default async function(pkgs: PackageRequirement[]) {
         if (program.includes("{{")) continue
 
         const pkgstr = utils.pkg.str(pkg)
-        const exec = `exec pkgx +${pkgstr} -- ${program} "$@"`
+        if (UNSAFE) {
+          const parts = pkgstr.split("/")
+          parts.pop()
+          await Deno.mkdir(Path.home().join(`.cache/pkgx/envs/${parts.join("/")}`).toString(), {recursive: true})
+        }
+        //FIXME: doing `set -a` clears the args env
+        const exec = UNSAFE ? undent`
```

Comment:
i'd probably alias the substitutions so you don't have to keep doing them:
```shell
  ENV_FILE=\"$\{XDG_CACHE_DIR:-$HOME/.cache\}/pkgx/envs/${pkgstr}.env\"
  PKGX_DIR=$\{PKGX_DIR:-$HOME/.pkgx\}
```

Reply:
Thought that but forgot at the last minute. Gonna add.